Entity/OfficeHours.py

```diff
@@ -1,10 +1,7 @@
-#OfficeHours
-
 class OfficeHours:
 
-   def __init__(self, Professors_id, ohroom, ohday, ohtime):
-      self.Professors_id = Professors_id
-      self.ohroom = ohroom
-      self.ohday = ohday
-      self.ohtime = ohtime
-      
+    def __init__(self, Professors_id, ohroom, ohday, ohtime):
+        self.Professors_id = Professors_id
+        self.ohroom = ohroom
```

Comment:
Can we add a comment for what these variables are? I had trouble figuring out from the name (despite the class being named 'OfficeHours'

Reply:
@snekiam, I agree! Please do feel empowered to make that change!